src/Driver/Driver.jl

```diff
@@ -300,6 +306,31 @@ function parse_commandline(
         metavar = "<horizontal>,<vertical>"
         arg_type = NTuple{2, Int}
         default = get_setting(:degree, defaults, global_defaults)
+        "--nelems"
+        help = "tuple of number of elements in each direction: 2 for GCM or 1 for single-stack (no space before/after comma)"
```

Comment:
This help string seems to say that a single element tuple is expected for single-stack, but the code seems to require a two element tuple?

Reply:
The option can either be a tuple of two elements for GCM or a single value for single stack.